src/v/coproc/pacemaker.cc

```diff
@@ -280,4 +280,13 @@ bool pacemaker::local_script_id_exists(script_id id) {
     return _scripts.find(id) != _scripts.end();
 }
 
+bool pacemaker::is_up_to_date() const {
+    return std::all_of(
+      _scripts.cbegin(),
+      _scripts.cend(),
+      [](const decltype(_scripts)::value_type& p) {
```

Comment:
is this preferred over using `const auto& p`?

Reply:
I thought we wanted to avoid using polymorphic lambdas

Reply:
unless someone can provide some evidence, perhaps we should allow them. also, when I put `const auto&` into cppinsights it becomes a template parameter (not polymorphic).